base/setup/usetup/usetup.c

```diff
@@ -3251,12 +3251,16 @@ IsValidPath(
     // TODO: Add check for 8.3 too.
 
     /* Path must be at least 2 characters long */
-//    if (Length < 2)
-//        return FALSE;
+    if (Length < 2)
+        return FALSE;
 
     /* Path must start with a backslash */
-//    if (InstallDir[0] != L'\\')
-//        return FALSE;
```

Comment:
These commented out lines were added in 9c64b57dc by Hermes, needs to be reviewed by him.

Reply:
These actually came from one of those commits shown in the blame there:
https://github.com/reactos/reactos/blame/14284ddc618661fc8c8e64acde2238ca3dfd7897/base/setup/usetup/filesup.c#L133

Reply:
The second commit by Eric is so descriptive...